packages/forms-web-app/src/views/appellant-submission/appeal-statement.njk

```diff
@@ -123,7 +123,9 @@
                 data-file-id="{{ appeal.yourAppealSection.appealStatement.uploadedFile.id }}"
                 data-file-location="{{ appeal.yourAppealSection.appealStatement.uploadedFile.location }}"
                 data-file-size="{{ appeal.yourAppealSection.appealStatement.uploadedFile.size }}">
-              {{- appeal.yourAppealSection.appealStatement.uploadedFile.name -}}
+              <a href="{{ ['/document', appeal.id, appeal.yourAppealSection.appealStatement.uploadedFile.id ] | join('/') }}" title="Download {{ appeal.yourAppealSection.appealStatement.uploadedFile.name }}" data-cy="appeal-upload-file-link">
```

Comment:
all the links follow this pattern of 

`[ 'build', 'link', 'here' ].join('/')`

I gave some thought to this and felt a custom filter or macro would be too specific and the layer of indirection didn't offer much. 

Doing it this way is more verbose, yet more customisable, so it seemed to be preferable.

Reply:
:+1: